en/docs/manage/api-management.md

```diff
@@ -0,0 +1,54 @@
+
+# Consumers
+
+The consumers' pane shows us the applications that have subscribed to your API.  You can manage access to your API by blocking or unblocking these applications through the Choreo Console. The Consumers pane lists the applications, and for each application, it shows the usage tier, owner, and status: blocked or unblocked. 
+
+# Documents
+
+API documentation is vital in helping API consumers understand the functionality of your API and is beneficial to marketing the API. The Choreo Console allows you to add a document on the fly by entering a name and the API contents. You can also preview the document and verify if it is suitable for publishing. 
+
+# Usage Plans
+
+You can secure your API backends by applying rate-limiting policies through the usage plans pane, typically in cases such as the following: 
+
+- To protect your APIs from common types of security attacks, such as certain types of denial of service (DOS) attacks
+- To regulate traffic according to infrastructure availability
+- To make an API, application, or resource available to a consumer at different levels of service, usually for monetization purpose
+
+Choreo allows you to apply a usage plan on the API that is not bound to an environment or build. Choreo supports the following usage plans by default:
+
+- Bronze - 1000 requests per minute
+- Silver - 2000 requests per minute
+- Gold - 5000 requests per minute
+- Unlimited - unlimited requests per minute
+
+# Permissions
+
+Permissions allow you to attach scopes to enable fine-grained access control to your API resources. You can create a scope entry by giving a scope name. You can then select and apply these scopes to the individual resources in your API. On the identity provider, a scope will have a role mapped to it. A role specifies the permissions associated with it. When you invoke the API resource with a specific scope, the access to your API resource is controlled based on the permissions associated with the role. 
+
+# Settings
+
+The Choreo Console allows you to define security, rate-limiting, and visibility settings for an API. You can configure the following settings: 
+
+- **Access mode** - Configure API access as internal or external. 
+- **API Visibility** - Configure API visibility as private or public.
+- **CORS configuration** - Enables CORS configurations and provides necessary CORS headers
+- **Application level security** - This determines the type of security used to safeguard this API.  You can secure your APIs with OAuth2, any other available security scheme, or both. If the OAuth2 option is selected, that API will need a valid OAuth2 token for successful invocation.
+- **Operations Configuration** - Configure rate limiting to your API in Choreo per API operation.
+
+# Properties
+
+You can view and edit the API properties, such as the business owner's name and email and the technical owner's name and email, from the Choreo console. 
```

Comment:
Suggestion: "You can view and edit the API properties (such as the business owner's name and email and the technical owner's name and email) from the Choreo Console."

Use title caps instead of sentence case for :"Choreo Console"